plugin/ims_lti/src/Service/Resource/LtiLineItemsResource.php

```diff
@@ -0,0 +1,266 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+use Chamilo\PluginBundle\Entity\ImsLti\LineItem;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+use Doctrine\ORM\TransactionRequiredException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException;
+use Symfony\Component\HttpKernel\Exception\UnsupportedMediaTypeHttpException;
+
+/**
+ * Class LtiLineItemsResource.
+ */
+class LtiLineItemsResource extends LtiAdvantageServiceResource
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class LtiLineItemsResource has a coupling between objects value of 16. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=4799654)